app/assets/stylesheets/welcome.css.scss

```diff
@@ -0,0 +1,26 @@
+#banner {
+	background-image: url(image_path('welcome.jpg'));
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position: top center;
+	width: 100%;
+	min-height: 500px;
+	display: table;
+	.banner_content {
+		display: table-cell;
+		vertical-align: middle;
+		text-align: center;
+		h1 {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3